spec/features/pets/delete_spec.rb

```diff
@@ -29,4 +29,28 @@
     expect(current_path).to eq("/shelters/#{@shelter.id}/pets")
     expect(page).to_not have_content("Penny")
   end
+
+  it "delete the pet and remove it from favorites" do
+
+    within("#nav") do
+      expect(page).to have_content("0")
+    end
+
+    visit "/pets/#{@penny.id}"
+    click_link "Add to Favorites"
+
+    within("#nav") do
+      expect(page).to have_content("1") 
+    end
+
+    click_link "Delete #{@penny.name}"
+
+    within("#nav") do
+      expect(page).to have_content("0")
```

Comment:
Started with no pets in favorites to show progression from zero to one, back to zero when we delete a favorited pet.